content/sips/sip-311.md

```diff
@@ -53,31 +53,52 @@ Teleporters involve adding `transferCrossChain` and `ccipReceive` functions to t
 
-We propose making the transfer function available on the snxUSD proxy, rather than the core system proxy. This is an extension of the ERC-20 interface (which already essentially includes a "same-chain transfer" function).
+We propose making the transfer function available on the snxUSD proxy, rather than the core system proxy. This is an extension of the ERC-20 interface (which already essentially includes a "same-chain" `transfer` function).
+
+#### Using CCIP for token transfers
+
+In addition to the new functions outlined in the technical specification below, this SIP also proposes that Chainlink's CCIP token pools be granted permission to mint and burn snxUSD in order to facilitate the cross-chain token transfers. This method allows for multiple benefits as described below:
+
+**Composability & Commercial Opportunity**
+
+Token transfers using the CCIP token pools provide Synthetix the opportunity, if desired, to open up Chainlink's private snxUSD Token Pool which would allow third parties to transfer snxUSD (even in combination with other tokens and/or data).
+
+**Enhanced security via the Active Risk Management (ARM) Network**
+
+With token transfers, the ARM Network has access to additional transaction information (i.e. token, amount, destination chain id) that enable it to perform additional risk management policies which cannot be implemented for arbitrary messaging. The ARM rule set will evolve over time into more advanced logic to identify and mitigate risk more effectively.
+
+**Enhanced security via Rate Limits**
+
+In contrast to arbitrary messaging, CCIP token transfers provide configurable rate limits per token pool per lane, which are set up in alignment with the token issuer. This feature is part of the heavily audited CCIP code base and only available for token transfers. CCIP token transfers also benefit from increased security via the global rate limits (across token pools), so not every token’s rate limit could be exploited at maximum in a worst case scenario.
+
+#### Fee Collection
 
 This proposal entails collecting fees in ETH, rather than snxUSD, for the following reasons:
 
 - Users will always receive the same amount of snxUSD that they have transferred.
 - This is the most efficient method to collect fees, as no additional token transfers are involved in the transaction.
 - Users are already paying ETH for gas fees to execute the transaction.
+- Reduces code complexity since the `CcipRouter` handles the fee calculation for the execution of the specified message.
+- Users are automatically refunded any left over ETH that was unused in the message execution.
```

Comment:
Suggest to clarify that this refund is specifically for the difference between the value that the user passes to transferCrossChain() and what CCIP Router returns as getFee().